cabal-install/Distribution/Client/CmdInstall.hs

```diff
@@ -658,13 +658,12 @@ installExes verbosity baseCtx buildCtx platform compiler
 
       mkFinalExeName :: UnqualComponentName -> FilePath
       mkFinalExeName exe = prefix <> unUnqualComponentName exe <> suffix <.> exeExtension platform
-      installdirUnknown path =
+      installdirUnknown =
         "installdir is not defined. Set it in your cabal config file "
-        ++ "or use --installdir=<path>. Using default installdir: " ++ show path
+        ++ "or use --installdir=<path>. Using default installdir: " ++ show installPath
 
   installdir <- fromFlagOrDefault
```

Comment:
this can be `let installdir = ... ` now, can't it. There are no effects happening?

Sorry for not spotting on the first look